runtime/onert/api/python/package/experimental/train/optimizer/optimizer.py

```diff
@@ -0,0 +1,15 @@
+from onert.native.libnnfw_api_pybind import trainable_ops
+
+
+class Optimizer:
+    """
+    Base class for optimizers. Subclasses should implement the `step` method.
+    """
```

Comment:
```suggestion
    Base class for optimizers.
```

Since `step()` is deleted, this needs to be updated.

Reply:
@ragmani But I think this has to be fixed. That's why I didn't approve this PR yet.

Reply:
Thanks, it's removed now :)